portfolio/src/test/java/com/google/sps/DataServletTest.java

```diff
@@ -79,11 +79,11 @@ public void tearDown() {
     return Arrays.asList(comments);
   }
 
-  private int getNumberOfEntriesInDatastore() {
+  int getNumberOfEntriesInDatastore() {
```

Comment:
Leave this, as is, for now (since we're doing this as an exercise) - but common code like this can be moved to a dedicated class, as well.

Reply:
ah ok-thanks for the feedback!